example/crs/mellanox.com_v1alpha1_nicclusterpolicy_cr_telemetry.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 NVIDIA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: mellanox.com/v1alpha1
+kind: NicClusterPolicy
+metadata:
+  name: nic-cluster-policy
+spec:
+  docaTelemetryService:
+    image: doca-telemetry
+    repository: nvcr.io/nvidia/doca
+    version: 1.15.5-doca2.5.0-host
```

Comment:
we generate updated examples from here : https://github.com/Mellanox/network-operator/tree/master/hack/templates/crs

so this needs to go into templates/crs

Reply:
I don't think we really need this template though - mostly kept it around for dev purposes. Will just remove.